lib/dogma/formatter/emacs.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Dogma.Formatter.Emacs do
```

Comment:
If we're not going for something general like `machine` then let's name it after Flycheck.